### ISSUE TITLE
resetNowPlaying does not hide the "remote control" in "iOS"

### ISSUE THREAD

**Issue by jcliang1212** [tanguyantoine/react-native-music-control#135]
# Description
I want to hide the "remote control" when the user press "home" button in "iOS", but the MusicControl.resetNowPlaying() does not work

1. Sample code (provide repo url or sample code)

none

2. Platform ?
- [x] iOS
- [ ] Android
   
3. Device
- [x] Simulator
- [x] Real device

**Comment by santiripper** [comment]
Same here on ios 11.2.5 iphone x, i'll test on other versions and devices.
Also you can`t update the nowplaying data

**Comment by tanguyantoine** [comment]
Could you share your repo ?

**Comment by parveen-lukha** [comment]
Facing same issue with resetNowPlaying() in IOS
It works perfect on Android but controls does not hide in IOS

**Comment by parveen-lukha** [comment]
I am using it with react-native-audio-toolkit and found that it does not hide control if my player is in playing state. So pausing it before reset helped in my case

**Comment by tanguyantoine** [comment]
any chance you provide an example?

**Comment by parveen-lukha** [comment]
@tanguyantoine 
My problem was that When I changed audio file I was always using 
MusicControl.setNowPlaying() //With metadata like title,artist of AudioFile
So on audio file change my player was preparing(Which takes time as its streaming)  the controls were irrelavent as player was not ready yet. Also it was showing data of previous audio file till player is prepared and setNowPlaying is called again. 
So on audioFile chage event I did:
1.  I reset the controls using MusicControl.resetNowPlaying();
2. After player is ready with new file I set controls using MusicControl.setNowPlaying();
This was creating issue when my player was in playing mode so the only thing I changed is 
I paused player before changing the audio
this.player.pause(err){
    if(err) {
      console.log(err)
   }
MusicControl.resetNowPlaying();
}
